internal/resources/providers/gcplib/inventory/provider_test.go

```diff
@@ -418,7 +474,7 @@ func (s *ProviderTestSuite) TestListLoggingAssets() {
 				return ""
 			},
 			getOrganizationDisplayName: func(_ context.Context, _ string) string {
-				return "OrganizationName1"
+				return ""
 			},
```

Comment:
this is the actual difference in the tests. the rest is just boilerplate to make test 2 outputs of `ListAllAssetTypesByName`